support-frontend/build.sbt

```diff
@@ -21,6 +20,7 @@ libraryDependencies ++= Seq(
   "com.amazonaws" % "aws-java-sdk-stepfunctions" % awsClientVersion,
   "com.amazonaws" % "aws-java-sdk-sts" % awsClientVersion,
   "com.amazonaws" % "aws-java-sdk-s3" % awsClientVersion,
+  "com.amazonaws" % "aws-java-sdk-lambda" % awsClientVersion,
```

Comment:
this was previously getting pulled in via support-models -> acquisition event producer -> acquisition value calculator client